src/predictors/ReLU.jl

```diff
@@ -300,7 +300,16 @@ ReLUQuadratic()
   └ moai_ReLU[2]*moai_z[2] = 0
 ```
 """
-struct ReLUQuadratic <: AbstractPredictor end
+struct ReLUQuadratic <: AbstractPredictor
+    relax_equality::Bool
+    relaxation_parameter::Float64
+    function ReLUQuadratic(;
+        relax_equality::Bool = false,
+        relaxation_parameter::Float64 = 0.0,
```

Comment:
I would make this
```suggestion
        relaxation_parameter::Union{Nothing,Float64} = nothing,
```
If `nothing`, we use `== 0`, otherwise, we assert `relaxation_parameter >= 0` and use `<= relaxation_parameter`.

Reply:
Yep, that's better.